terraform/revisions.tf

```diff
@@ -0,0 +1,22 @@
+locals {
```

Comment:
missing renovate tag?

Reply:
there's a [fup](https://warthogs.atlassian.net/browse/DPE-5673) task for implementing this on the update bundle workflow (actually independent from the renovate)